.github/workflows/CD.yaml

```diff
@@ -123,10 +123,17 @@ jobs:
           PROXY_PRIMARY_URL: ${{ secrets.PROXY_PRIMARY_URL }}
           PROXY_LETSENCRYPT_ARGS: ${{ secrets.PROXY_LETSENCRYPT_ARGS }}
           RESET_BACKEND_DATABASE_AND_REPOPULATE: ${{ github.event.inputs.resetBackendAndRepopulate }}
+          RABBITMQ_USER: ${{ secrets.RABBITMQ_USER }}
```

Comment:
What Values are set for the different environments? 
The Values for Preview are not yet set at all.
Where is this all documented?

Reply:
The samve values are used for all CD Environments, see [https://github.com/d-fine/DatalandInternal/wiki/Server](https://github.com/d-fine/DatalandInternal/wiki/Server); CI uses the same env values as the local development environment, see CI.yaml or setEnvironmentVariables.ps1.

Reply:
ok